### DISCUSSION TITLE
cn/2024/12/07/bearblog/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#192] [Announcements]
# cn/2024/12/07/bearblog/

https://hongtaoh.com/cn/2024/12/07/bearblog/

**Comment by toyo2333** [comment]
可以考虑hugo bear 这主题，应该符合你的需求：https://github.com/janraasch/hugo-bearblog/

**Comment by CyrusYip** [comment]
> 说实话我目前还没有看到一个免费的、面对中文群体的、美观的、易用的博客系统。如果有，请告知我一声。

Publii 可能符合你的要求。这里有篇介绍文：[Publii：静态博客的另一个选择 - 印记](https://yinji.org/5251.html)。

**Reply by hongtaoh** [reply]
你这个链接我打不开。另外，我看了看 Publii 依然是一个需要下载的工具，和我想要的并不是一回事。我想要的是一个平台。

**Reply by CyrusYip** [reply]
我认为没有平台（在线服务）可以满足免费这个要求。网站是运行在服务器上的，服务器是要花钱租的，流量高的平台倒是可以考虑加广告来盈利。小平台的话做成 Bearblog 这种月租模式比较靠谱。

**Comment by shenweiyan** [comment]
用 github issues 或者 discussions
